### PR TITLE
Add configuration of reactive slacks repartition

### PR DESCRIPTION
**Pull request by p-arvy**
…ode.

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

#20 

**What kind of change does this PR introduce?**

The high limit on reactive slack variables is removed. Slacks are now at each node of the network.

**What is the current behavior?**

The reactive slack variables have upper bound = 500 MVar. Only buses without generator, vsc and svc have reactive slacks.

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

**Review by klorel — COMMENTED** [review]
please change the value of buses_with_reactive_slacks to have an explicit description of the behavior

**Review by klorel — APPROVED** [review]
AMPL CODE OK

File: open-reac/src/test/resources/expected_inputs/param_buses_with_reactive_slack.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#amplId powsyblId
```
Comment (annetill) [inline open-reac/src/test/resources/expected_inputs/param_buses_with_reactive_slack.txt:1]: powsyblId :-) everything here is powsybl

Comment (p-arvy) [reply open-reac/src/test/resources/expected_inputs/param_buses_with_reactive_slack.txt:1]: I agree with this. We have to change the write function of AbstractElementInput class then :)
"#num id" would be consistent with other input/output files. Do you agree ?

File: open-reac/src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java [hunk 2]
```diff
@@ -190,6 +200,15 @@ public OpenReacParameters setMaxPlausibleHighVoltageLimit(double maxPlausibleHig
         return this;
     }
 
+    public OpenReacBusesWithReactiveSlackConfig getBusesWithReactiveSlackConfig() {
```
Comment (annetill) [inline open-reac/src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java:203]: Java doc missing.

File: open-reac/src/test/resources/expected_inputs/param_buses_with_reactive_slack.txt (hunk 1 above)
Comment (annetill) [reply open-reac/src/test/resources/expected_inputs/param_buses_with_reactive_slack.txt:1]: If it is in core, it is too late...

**Review by annetill — APPROVED** [review]

**Review by annetill — APPROVED** [review]

**Pull request merged**